app/views/markets/_header.html.erb

```diff
@@ -2,7 +2,7 @@
   <div class="container-fluid">
     <ul id="menu" class="nav navbar-nav navbar-left">
       <a class="navbar-brand" href="/">
-        <%= ENV.fetch('TITLE') %>
+        <%= t('platform_title') %>
```

Comment:
@dpaluy I don't think this is good idea to put <%= ENV.fetch('TITLE', 'Peatio') %> into each locale. It is much better to use ENV.fetch('TITLE', 'Peatio') unless translation exists.

Reply:
The best practise is setting text in dedicated config file or locale.yml
Pay attention that the project title is defined by customer based on their language requirements. 
You can't set it in `ENV`, unless you have a single language support